isamples_metadata/SmithsonianTransformer.py

```diff
@@ -196,8 +202,8 @@ def produced_by_responsibilities(self) -> list[dict[str, str]]:
         self._add_to_responsibilities("scientificNameAuthorship", "scientific name authorship", responsibilities)
 
         # unfortunately it looks like this field uses a ; separator for multiple people
-        identified_by = self.source_record["identifiedBy"]
-        if len(identified_by) > 0:
+        identified_by = self.source_record.get("identifiedBy")
+        if identified_by is not None and len(identified_by) > 0:
```

Comment:
These were causing crashes for me running locally -- not sure why we hadn't seen them before, but I went ahead and fixed it since it was straightforward and safe.